### DISCUSSION TITLE
Any way to run the firejail with any other name? (EAC issue)

### DISCUSSION THREAD

**Discussion by hashelq** [netblue30/firejail#6167] [Q&A] [answered]
It seems EAC finally started detecting firejail, is there a builtin way to run the firejail lets say with name 'hello-embark'? I tried ln-s, cp etc, but it does not seem to work on both nixos/alpine I tested at.

**Comment by kmk3** [comment]
> It seems EAC finally started detecting firejail,

What does it say?  Are you sure that it fails because of the name and not
because it cannot access some path?

Does it fail only when running under firejail or by just having it installed at
all?

> is there a builtin way to run the firejail lets say with name 'hello-embark'?
> I tried ln-s, cp etc, but it does not seem to work on both nixos/alpine I
> tested at.

Probably not, as many paths are hardcoded, including /run/firejail.

It might be possible to make them configurable when building, but it would not
be trivial, as they are hardcoded in many places.  For example, in the code,
man pages, tests, profiles and contrib files.

**Comment by rusty-snake** [comment]
> It seems EAC finally started detecting firejail, is there a builtin way to run the firejail lets say with name 'hello-embark'?

I only find Windows binaries, do you run it under wine?

Anyway. If as software detects firejail personally rather than a generic check for any sandbox/container software, it will do so by checking `$container` or `/run/firejail` paths (maybe in mountinfo). This can all be spoofed. However, a "correct" implementation would check for a sandbox/container which can always be fingerprinted.

**Comment by hashelq** [comment, marked as answer]
NVM about the game, it seems the game was complaining about proton version, not firejail.
(I also found a way to change process firejail's name not to appear in ps/htop/proc so glowing by rbinding to /proc/1/cmdline)
